### PR TITLE
move config json file to values

### PR DESCRIPTION
**Pull request by nathwang**
move config json file to values

### CODE REVIEW

File: charts/fleet-telemetry/README.md [hunk 1]
```diff
@@ -57,3 +58,41 @@ tlsSecret:
     value of the cert PEM
   tlsKey: |
     value the private key
+config:
+  data: |
```
Comment (aaronpkahn) [inline charts/fleet-telemetry/README.md:62]: can we optionally reference a file?

Comment (nathwang) [reply charts/fleet-telemetry/README.md:62]: no, helm chart doesn't support that, client can shell script to do it

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -9,7 +9,18 @@ on:
...
 
 jobs:
+  validate-input:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate input
+        id: validate
+        run: |
+          if [[ "${{ github.event.inputs.version_bump }}" != "major" && "${{ github.event.inputs.version_bump }}" != "minor" && "${{ github.event.inputs.version_bump }}" != "patch" ]]; then
+            echo "Invalid input for version_bump. Please provide 'major', 'minor', or 'patch'."
```
[the remark this answers is not in the document]
Comment (nathwang) [reply .github/workflows/release.yml:19]: this is from your code, I just revert it back

File: charts/fleet-telemetry/README.md [hunk 3]
```diff
@@ -57,3 +60,99 @@ tlsSecret:
...
+          "logger"
+        ],
+        "errors": [
+          "logger"
+        ],
+        "V": [
+          "logger"
+        ]
+      },
+      "use_default_eng_ca": true,
```
Comment (aaronpkahn) [inline charts/fleet-telemetry/README.md:94]: Please remove this key or set value to `false` as it should be rare to run the app in engineering mode.

File: .github/workflows/release.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (agbpatro) [reply .github/workflows/release.yml:19]: I think we don't need this altogether

File: charts/fleet-telemetry/README.md (hunk 3 above)
Comment (nathwang) [reply charts/fleet-telemetry/README.md]: done

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -4,9 +4,8 @@ on:
   workflow_dispatch:
     inputs:
       version_bump:
-        description: 'Version bump (major, minor, patch)'
+        description: 'Version bump'
         required: true
```
Comment (agbpatro) [inline .github/workflows/release.yml:8]: Since there are no inputs, you can simply do:-
```sh
on:
  workflow_dispatch:
```

**Review by agbpatro — APPROVED** [review]

**Pull request merged**